src/components/radio-button/radio-button.types.ts

```diff
@@ -0,0 +1,12 @@
+import { StyleProp, TextStyle, ViewStyle } from 'react-native';
+
+export interface RadioButtonProps {
+  value: string;
+  selected: boolean;
+  onPress: (value: string) => void;
+  label?: string;
+  disabled?: boolean;
+  kind?: 'primary' | 'error' | 'success';
+  containerStyle?: StyleProp<ViewStyle>;
+  labelStyle?: StyleProp<TextStyle>;
```

Comment:
We do not want consumer to pass style for container and label as it would lead to defeating the purpose of standardizing the look and feel.